.buildkite/pipelines/emergency_release.yml

```diff
@@ -1,3 +1,20 @@
+env:
+  VERSION: ${BUILDKITE_COMMIT:0:12}
+  DOCKER_IMAGE_TAG: git-${VERSION}
+  DOCKER_IMAGE: docker.elastic.co/kibana-ci/kibana-serverless
+  EMERGENCY_RELEASE_REMOTE_SERVICE_CONFIG: https://raw.githubusercontent.com/elastic/serverless-gitops/main/gen/gpctl/kibana/emergency.yaml
+
 steps:
-  - label: Placeholder
-    command: echo "Hello!"
+  - label: ':label: Create tag'
+    command: .buildkite/scripts/serverless/create_deploy_tag/create_fix_tag.sh
+    if: build.branch =~ /^deploy-fix@.*\$/
+
+  - wait
+
+  - label: ':argo: Emergency Release'
+    if: build.tag =~ /^deploy-fix@.*\$/ && build.branch =~ /^deploy-fix@.*\$/
```

Comment:
If the tag is only created in the previous step, this might never be true, because it's evaluated at the time of the pipeline upload. Maybe the tag condition is not needed?

Source: https://buildkite.com/docs/pipelines/conditionals#conditionals-in-pipelines
> Since if conditions are evaluated at the time of the pipeline upload, it's not possible to use the if attribute to conditionally run a step based on the result of another step.

Reply:
I pushed https://github.com/elastic/kibana/pull/176258/commits/0d3c94e755bbe41eaff8d244658652d6d5642b7d.  Makes sense, we can rely on the previous step as confirmation the tag exists.